server/api/cart.js

```diff
@@ -0,0 +1,94 @@
+const router = require('express').Router()
+const {Cart, CartProducts, Product} = require('../db/models')
+module.exports = router
+
+// GET /api/cart/:id
+router.get('/:id', async (req, res, next) => {
```

Comment:
How would the frontend know what the cardId is? Is it more likely you're searching for a cart by a cartId or by a userId?